flash/core/model.py

```diff
@@ -99,8 +99,7 @@ def step(self, batch: Any, batch_idx: int) -> Any:
         y_hat = self.to_metrics_format(y_hat)
         for name, metric in self.metrics.items():
             if isinstance(metric, torchmetrics.metric.Metric):
-                metric(y_hat, y)
-                logs[name] = metric  # log the metric itself if it is of type Metric
+                logs[name] = metric(y_hat, y)  # log the metric itself if it is of type Metric
```

Comment:
Not logging the metric any more so the comment should change. Is this to do with https://github.com/PyTorchLightning/pytorch-lightning/issues/7052 ? If so, maybe add a todo to change this back once it works again?

Reply:
Changed to what it was before.